src/org/labkey/test/util/core/webdav/WebDavUploadHelper.java

```diff
@@ -133,6 +133,37 @@ public void uploadFile(@NotNull File file)
         uploadFile(file, "");
     }
 
+    public void deleteFile(@NotNull String file, boolean shouldFail)
+    {
+        try
+        {
+            if (_sardine.exists(file))
```

Comment:
I probably wouldn't check if it exists here, as the caller may be wanting to test what the server does when you try deleting a file that doesn't exist.